src/client/cli/formatter/csv_formatter.cpp

```diff
@@ -85,6 +85,23 @@ std::string mp::CSVFormatter::format(const FindReply& reply) const
 {
     fmt::memory_buffer buf;
 
+    fmt::format_to(buf, "Blueprint,Remote,Aliases,OS,Release,Version\n");
+
+    for (const auto& blueprint : reply.blueprints_info())
```

Comment:
This is a bit tricky to output in `csv` format as sheets aren't supported. The best adhoc solution that I could find was to simply add a newline between the two bodies of data.